pom.xml

```diff
@@ -178,6 +178,7 @@
         <graphql-spqr.version>0.0.4</graphql-spqr.version>
         <gridgain.version>6.6.9</gridgain.version>
         <guava.version>28.1-jre</guava.version>
+        <xe-rates-client.version.version>0.1.0</xe-rates-client.version.version>
```

Comment:
why is it called xe-rates-client.version.version????

Reply:
Why is it not ordered alphabetically???

Reply:
Fixed